webpack.config.js

```diff
@@ -12,11 +12,11 @@ const config = {
   module: {
     loaders: [
       {
-        test: /\.js$/,
+        test: /\.jsx?$/,
```

Comment:
I just merged these two original loaders.

BTW, your so-said es6 file in this repo are all written in jsx and not pure ES2015.